notebooks/idr0062_prediction.ipynb

```diff
@@ -454,6 +481,25 @@
     "conn.close()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d2b111fe",
+   "metadata": {},
+   "source": [
+    "## Exercises\n",
+    "\n",
+    "**Exercise 1:**\n",
+    " - Read the geojson file.\n",
```

Comment:
Maybe

``Using the json library, read teh geojson file into a variablel in this notebook.``

I think we need to offer a bit more lead to give a realistic chance to succeed with the exercise (without looking at the Solution)